<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HistoriqueVaccinDialog</class>
 <widget class="QDialog" name="HistoriqueVaccinDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>525</width>
    <height>285</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTableWidget" name="tableWidgethis">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>431</width>
     <height>171</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtontxtstock">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>210</y>
     <width>191</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>fichier historique stock</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ButtonExit">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>210</y>
     <width>81</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
